Lib/unittest/test/testmock/testmock.py

```diff
@@ -1271,6 +1286,20 @@ def test_assert_called_once(self):
         with self.assertRaises(AssertionError):
             m.hello.assert_called_once()
 
+    def test_assert_called_once_message(self):
+        m = Mock()
+        m(1, 2)
+        m(3)
+        self.assertRaisesRegex(AssertionError,
+            r"Calls: \[call\(1, 2\), call\(3\)\]",
+            m.assert_called_once)
+
+    def test_assert_called_once_message_not_called(self):
+        m = Mock()
+        with self.assertRaises(AssertionError) as e:
+            m.assert_called_once()
+        self.assertFalse("Calls:" in str(e.exception))
```

Comment:
`self.assertNotIn("Calls:", str(e.exception))` will give a nicer error message if it fails.

Reply:
Thanks.